From b6ad961e98ed3bac69bd3f02415bf9791437fa5d Mon Sep 17 00:00:00 2001
From: Xiaowen Liu <b37945@freescale.com>
Date: Tue, 2 Apr 2013 15:08:05 +0800
Subject: [PATCH 440/635] ENGR00255297 fix format is recognized as YUY2 not
 supported for usb camera.

remove the YUY2 format reporting to framework.
then, it is ok to do CTS Verifier for usb camera.

Signed-off-by: Xiaowen Liu <b37945@freescale.com>
---
 mx6/libcamera2/UvcDevice.cpp | 1 -
 1 file changed, 1 deletion(-)

diff --git a/mx6/libcamera2/UvcDevice.cpp b/mx6/libcamera2/UvcDevice.cpp
index 37a38a6..a9bd6d0 100644
--- a/mx6/libcamera2/UvcDevice.cpp
+++ b/mx6/libcamera2/UvcDevice.cpp
@@ -46,7 +46,6 @@ void UvcDevice::adjustSensorFormats(int *src, int len)
                 break;
 
             case v4l2_fourcc('Y', 'U', 'Y', 'V'):
-                mSensorFormats[k++] = HAL_PIXEL_FORMAT_YCbCr_422_I;
                 if (mDefaultFormat == 0) {
                     mDefaultFormat = HAL_PIXEL_FORMAT_YCbCr_422_I;
                 }
-- 
1.8.0

